shared/html/index.html

```diff
@@ -28,7 +28,7 @@
   <!-- Default footer. -->
   <footer>
     <a target="_blank" href="http://groups.csail.mit.edu/haystack/">Haystack group</a> at <a target="_blank" href="http://csail.mit.edu/">MIT CSAIL</a> &amp; <a target="_blank"
-    href="http://bocoup.com">Bocoup</a> collaboration - <a target="_blank" href="http://github.com/haystack/tipsy">Source</a> / <a target="_blank" href="http://tipsy.csail.mit.edu">Project</a> - MIT License 2015
+    href="http://bocoup.com">Bocoup</a> collaboration – <a target="_blank" href="http://github.com/haystack/tipsy">Source</a> / <a target="_blank" href="http://tipsy.csail.mit.edu">Project</a> – MIT License 2017
```

Comment:
Shouldn't this be 2015-2017? Ref: https://www.gnu.org/licenses/gpl-howto.en.html (which reasoning should still be transferable to here ...).